troposphere/static/js/components/NotFoundPage.react.js

```diff
@@ -10,7 +10,7 @@ export default React.createClass({
 
         return (
         <div className="container">
-            <h2>Page Unavailable</h2>
+            <h1 className="t-display">Page Unavailable</h1>
```

Comment:
I would imagine that this could be included in the `troposphere-ui` as part of the style guide? Or is that not a good place for it?